Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -3478,15 +3550,23 @@ def main():
     """
     PARSE AND VALIDATE INTEGRATION PARAMS
     """
+
     command = demisto.command()
     demisto.debug(f"Executing command {command}")
 
     params = demisto.params()
     args = demisto.args()
     verify = not params.get("insecure", False)
     use_oauth = params.get("use_oauth", False)
+    use_jwt = params.get("use_jwt", False)
     oauth_params = {}
 
+    # use jwt only with OAuth
+    if use_jwt and use_oauth:
+        raise ValueError("choose only one authentication method (OAuth ot JWT)")
+    elif use_jwt and not use_oauth:
```

Comment:
In this case, for sure the use_oauth will be false.

```suggestion
    elif use_jwt:
```